frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/project/MacheteProjectResolver.java

```diff
@@ -55,15 +52,17 @@ public boolean cancelTask(ExternalSystemTaskId taskId, ExternalSystemTaskNotific
     return false;
   }
 
-  @SuppressWarnings({"guieffect:call.invalid.ui", "interning:not.interned"})
+  @SuppressWarnings("interning:not.interned")
   private Boolean hasSelectedGitMacheteTab(Project project) {
     var toolWindowManager = ToolWindowManager.getInstance(project);
     var toolWindow = toolWindowManager.getToolWindow(ToolWindowId.VCS);
     if (toolWindow != null) {
-      var contentManager = toolWindow.getContentManager();
```

Comment:
Whoops okay so this was potentially unsafe?

Reply:
:yes: